### PR TITLE
Adds features to check crypto-related hacks

### PR DESCRIPTION
**Pull request by jose-donato**
# Description
This PR adds [rekt.news](https://rekt.news) source and one feature related to crypto hacks to `overview` crypto menu:
* `ch`: displays crypto related hacks
image

* `ch --slug polynetwork-rekt`: displays the history of a certain crypto hack
image

# How has this been tested?
Tested feature in jupyter notebook and after in the terminal.

# Checklist:

- [x] Update [our Hugo documentation](https://gamestonkterminal.github.io/GamestonkTerminal/) following [these guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/scripts).

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [x] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.

### CODE REVIEW

**Comment by jmaslek** [comment]
Can you either have a default on ich or let the user know they need to supply one instead of just listing everyyyything.

```
2022 Jan 21, 09:15 (✨) /crypto/ov/ $ ich
Slug "None" not found, try one of the following: polynetwork-rekt,bitmart-rekt,compound-rekt,vulcan-forged-rekt,cream-rekt-2,badger-rekt,ascendex-rekt,easyfi-rekt,uranium-rekt,bzx-rekt,pancakebunny-rekt,epic-hack-homie,alpha-finance-rekt,veefinance-rekt,cryptocom-rekt,meerkat-finance-bsc-rekt,monox-rekt,spartan-rekt,grim-finance-rekt,stablemagnet-rekt,paid-rekt,harvest-finance-rekt,xtoken-rekt,popsicle-rekt,pickle-finance-rekt,cream-rekt,snowdog-rekt,bearn-rekt,indexed-finance-rekt,eminence-rekt-in-prod,furucombo-rekt,deathbed-confessions-c3pr,value-rekt3,yearn-rekt,arbix-rekt,rari-capital-rekt,value-rekt2,cover-rekt,punkprotocol-rekt,visor-finance-rekt,thorchain-rekt2,hack-epidemic,lcx-rekt,anyswap-rekt,warp-finance-rekt,burgerswap-rekt,value-defi-rekt,alchemix-rekt,belt-rekt,bondly-rekt,roll-rekt,thorchain-rekt,xtoken-rekt-x2,11-rekt,chainswap-rekt,daomaker-rekt,jaypegs-automart-rekt,pancakebunny2-rekt,au-dodo-rekt,akropolis-rekt,bent-finance,8ight-finance-rekt,levyathan-rekt,the-big-combo,autoshark-rekt,merlinlabs-rekt,merlin2-rekt,merlin3-rekt,saddle-finance-rekt,safedollar-rekt
```

File: gamestonk_terminal/cryptocurrency/overview/overview_controller.py [hunk 1]
```diff
@@ -83,6 +87,8 @@ def __init__(self, queue: List[str] = None):
 
         if session and gtff.USE_PROMPT_TOOLKIT:
             choices: dict = {c: {} for c in self.controller_choices}
+            choices["ch"]["--sort"] = {c: None for c in rekt_model.HACKS_COLUMNS}
```
Comment (jmaslek) [inline gamestonk_terminal/cryptocurrency/overview/overview_controller.py:90]: should also add `choices["ch"]["-s"]`

**Comment by DidierRLopes** [comment]
Sooo, why not merging these 2 commands?

E.g. having `ch` as is, but if user specifies a slug then we go to the specific. It's much simpler and intuitive, this is what we do with news articles!

**Comment by jose-donato** [comment]
> Sooo, why not merging these 2 commands?
> 
> E.g. having `ch` as is, but if user specifies a slug then we go to the specific. It's much simpler and intuitive, this is what we do with news articles!

yea this is actually way more clever 😄 will do it

**Comment by jose-donato** [comment]
> Sooo, why not merging these 2 commands?
> 
> E.g. having `ch` as is, but if user specifies a slug then we go to the specific. It's much simpler and intuitive, this is what we do with news articles!

done

> Sooo, why not merging these 2 commands?
> 
> E.g. having `ch` as is, but if user specifies a slug then we go to the specific. It's much simpler and intuitive, this is what we do with news articles!

should be good now

**Review by DidierRLopes — COMMENTED** [review]
last request, I'm a big fan of being able to do `cmd parameter` and not needing any flag. In this case it makes sense for this to happen with the `slug` can we make it happen? 😬

**Comment by jose-donato** [comment]
> last request, I'm a big fan of being able to do `cmd parameter` and not needing any flag. In this case it makes sense for this to happen with the `slug` can we make it happen? 😬

done

**Review by DidierRLopes — APPROVED** [review]

**Pull request merged**